{
    "ban": {
        "name": "ban",
        "description": "Bans an user",
        "options": {
            "user": {
                "name": "user",
                "description": "The user to be banned"
            },
            "del_hours": {
                "name": "delete_hours",
                "description": "Hours of messages to delete"
            },
            "reason": {
                "name": "reason",
                "description": "The ban reason"
            }
        },
        "caller": {
            "permission_error": "You are missing the \"Ban members\" permission.",
            "interact_error": "You cannot ban {mention}."
        },
        "bot": {
            "permission_error": "The bot is missing the \"Ban members\" permission.",
            "interact_error": "The bot cannot ban {mention}."
        },
        "ban_status": "Banning...",
        "success": "{bannedUser} was banned successfully for the reason '{reason}', and {delHours} {delHours, choice, 1#hour|2<hours} of messages were deleted",
        "unknown_error": "An unknown error occurred"
    },
    "exception": {
        "name": "exception",
        "description": "Voluntarily throws an exception"
    }
}